applications/nrf_desktop/doc/smp.rst

```diff
@@ -3,10 +3,10 @@
 Simple Management Protocol module
 #################################
 
-Use the Simple Management Protocol module to perform DFU over Bluetooth LE.
+Use the Simple Management Protocol module to perform a Device Firmware Upgrade (DFU) over Bluetooth Low Energy.
 
 The DFU can be performed among others using the nRF Connect Android application.
-The ``DFU`` button appears in the Bluetooth connected device's tab on Android device.
+The ``DFU`` button appears in the *Bluetooth connected devices* tab on the Android device.
```

Comment:
I meant that the button is visible in the tab representing given device. Rephrased,